test/reorderError1.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit Main Test Suite</title>
+	<link rel="stylesheet" href="../dist/qunit.css">
+	<script src="../dist/qunit.js"></script>
+	<script>
+	// When failed test that checked test call count.
+	// Ref #907
+	// This issues has 2 case.
+	// First Case.
+	// 1. First test failed and second test successed in first module.
+	// 2. First test failed in second module.
+	// 3. Test count should be 3.
+
+	if ( window.sessionStorage ) {
```

Comment:
Comments should end with no blank line after but be preceded by one. Cut the blank line at [line 16](https://github.com/jquery/qunit/pull/907/files#diff-990591cfdcf8732c674cc9f9613d4356R16) and add a blank line before [line 9](https://github.com/jquery/qunit/pull/907/files#diff-990591cfdcf8732c674cc9f9613d4356R9)